src/components/Header/Header.js

```diff
@@ -1,14 +1,49 @@
-import './Header.css'
+import styled from 'styled-components'
```

Comment:
maybe change to:
`import styled from 'styled-components/macro';`

Reply:
```suggestion
import styled from 'styled-components/macro'
```